spec/serializers/topic_view_serializer_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
```suggestion
```

Minor but this is no longer required going forward since we've added this to `.rspec`